Sources/_CryptoExtras/RSA/RSA_boring.swift

```diff
@@ -101,6 +101,10 @@ extension BoringSSLRSAPrivateKey {
     internal func decrypt<D: DataProtocol>(_ data: D, padding: _RSA.Encryption.Padding) throws -> Data {
         return try self.backing.decrypt(data, padding: padding)
     }
+
+    internal func blindSignature<D: DataProtocol>(_ blindedMessage: D) throws -> _RSA.BlindSigning.BlindSignature {
```

Comment:
If we're aligning with public API for signatures, we would use `blindSignature(for message)`

As far as the signer goes, they do a blind sign operation, but they don't know what they sign, so I would just call it the message, not blindedMessage

Reply:
Can I confirm if this comment pertains to only the parameter name or to the type. There’s some other discussion about whether this takes its own `BlindedMessage` type rather than arbitrary data.

Reply:
Done: the public API and all the internal functions are now `blindSignature(for message:)`.